src/lib/operators/table_scan.hpp

```diff
@@ -48,6 +48,28 @@ class TableScan : public AbstractReadOnlyOperator {
    */
   std::vector<ChunkID> excluded_chunk_ids;
 
+  struct TableScanPerformanceData : public OperatorPerformanceData<AbstractOperatorPerformanceData::NoSteps> {
+    size_t chunk_scans_skipped{0};
+    size_t chunk_scans_sorted{0};
+
+    void output_to_stream(std::ostream& stream, DescriptionMode description_mode) const {
+      const auto separator = description_mode == DescriptionMode::MultiLine ? "\n" : " ";
```

Comment:
Feel free to spin this off into a new issue: It feels like our lives would be easier if we simply passed in the separator.

Reply:
Well, not necessarily. If you look into the PQP tooltip, where we use a multiline output of the OperatorPerformanceData, there is a difference (not necessarily, but anything else is major ugly) between a simple separator and printing list items.

Reply:
ok